eng/helix/helix.proj

```diff
@@ -47,6 +49,15 @@
     <AdditionalDotNetPackage Include="$(MicrosoftNETCoreBrowserDebugHostTransportVersion)">
       <PackageType>runtime</PackageType>
     </AdditionalDotNetPackage>
+
+    <!-- Grab the tool packages for what HelixTestRunner installs. -->
+    <HelixCorrelationPayload Include="$(NUGET_PACKAGES)\dotnet-dump\$(DotnetDumpVersion)\dotnet-dump.$(DotnetDumpVersion).nupkg" />
+    <HelixCorrelationPayload Include="$(NUGET_PACKAGES)\dotnet-ef\$(dotnetefVersion)\dotnet-ef.$(dotnetefVersion).nupkg" />
+    <HelixCorrelationPayload Include="$(NUGET_PACKAGES)\dotnet-serve\$(DotnetServeVersion)\dotnet-serve.$(DotnetServeVersion).nupkg" />
+
+    <!-- Grab published `HelixTestRunner` project output. -->
+    <HelixCorrelationPayload Include="$(ArtifactsBinDir)HelixTestRunner\$(Configuration)\net7.0\publish\"
```

Comment:
Maybe easier to just copy the publish directory into the workitem rather than using correlationpayloads for the helix test runner, its not like a huge publish dir right?  Especially if the same dll would work on all queues since I'm assuming we don't have any plat specific code in the runner

Reply:
I disagree because including the files in `<HelixContent />` embiggens hundreds of work item payloads rather than the 3-11 correlation payloads. This also keeps things together w/ the other platform-neutral / non-test-specific bits like the tool packages above and the App.Ref / App.Runtime (non-`crossgen`ed) payloads near the end of this file. I also don't see anything "simpler" about your proposal. Am I missing something❔

----

In fact, I'm tempted to move all but `@(HelixContent)` from https://github.com/dotnet/aspnetcore/blob/f6cddc1136fa9310a8fa0a06af8a461b409071a9/eng/targets/Helix.targets#L79-L91 into a correlation payload if only to reduce the number of times those small files are duplicated in `publish` directories and then work item payloads. Not an immediate problem but does cost a bit.

Somewhat separately there's no reason not to move https://github.com/dotnet/aspnetcore/blob/f6cddc1136fa9310a8fa0a06af8a461b409071a9/eng/targets/Helix.props#L27 into the correlation payloads.

Reply:
Okay I leave it to you, this part of the infrastructure has evolved very far from where it started